### PR TITLE
net-proxy/proxy-server: Add proxy-server:6.0.0

### PR DESCRIPTION
**Pull request by Jackarain**

### CODE REVIEW

File: net-proxy/proxy-server/proxy-server-6.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+LICENSE="Boost-1.0"
+SLOT="0"
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+BDEPEND=""
+
+src_configure() {
+	cmake_src_configure
```
Comment (Universebenzene) [inline net-proxy/proxy-server/proxy-server-6.0.0.ebuild]: This should be the default behavior of `src_configure` when `cmake.eclass` is inherited. No need to be written out.

File: net-proxy/proxy-server/proxy-server-6.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+src_compile() {
+	cmake_src_compile
```
Comment (Universebenzene) [inline net-proxy/proxy-server/proxy-server-6.0.0.ebuild]: Similar as `src_configure`

File: net-proxy/proxy-server/proxy-server-6.0.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+src_install() {
+	cmake_src_install
```
Comment (Universebenzene) [inline net-proxy/proxy-server/proxy-server-6.0.0.ebuild]: Similar as `src_configure`

File: net-proxy/proxy-server/proxy-server-6.0.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit cmake
+
+DESCRIPTION="Implementation of all proxy protocols using modern c++"
+HOMEPAGE="https://github.com/jackarain/proxy"
+SRC_URI="https://github.com/Jackarain/proxy/archive/refs/tags/v${PV}.tar.gz -> ${PV}.tar.gz"
```
Comment (Universebenzene) [inline net-proxy/proxy-server/proxy-server-6.0.0.ebuild]: Better renamed as `${P}.tar.gz`

File: net-proxy/proxy-server/metadata.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+<email>jack.wgm@gmail.com</email>
+<name>Jackarain</name>
+</maintainer>
```
Comment (Universebenzene) [inline net-proxy/proxy-server/metadata.xml:7]: Add a remote id as the pkgcheck says

File: net-proxy/proxy-server/proxy-server-6.0.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+src_compile() {
+	cmake_src_compile
+}
+
+src_install() {
+	cmake_src_install
+}
+
```
Comment (Universebenzene) [inline net-proxy/proxy-server/proxy-server-6.0.0.ebuild]: Blank line not needed at the end.

**Comment by Jackarain** [comment]
pkgcheck 通过了, 应该没问题了

**Review by Universebenzene — APPROVED** [review]

File: net-proxy/proxy-server/proxy-server-6.0.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit cmake
+
+DESCRIPTION="Implementation of all proxy protocols using modern c++"
+HOMEPAGE="https://github.com/jackarain/proxy"
+SRC_URI="https://github.com/Jackarain/proxy/archive/refs/tags/v${PV}.tar.gz -> proxy-${PV}.tar.gz"
```
Comment (liangyongxiang) [inline net-proxy/proxy-server/proxy-server-6.0.0.ebuild:9]: 这里推荐重命名成 proxy-server

**Pull request merged**